src/components/data-processing/CellSizeDistribution/CellSizeDistribution.jsx

```diff
@@ -512,7 +512,13 @@ class CellSizeDistribution extends React.Component {
             <Space direction='vertical' style={{ width: '100%' }} />
             <Collapse defaultActiveKey={['filtering-settings']}>
               <Panel header='Filtering Settings' collapsible={!filtering ? 'disabled' : 'header'} key='filtering-settings'>
-                <CalculationConfig experimentId={experimentId} sampleId={sampleId} plotType='bin step' sampleIds={sampleIds} />
+                <CalculationConfig
+                  experimentId={experimentId}
+                  sampleId={sampleId}
+                  plotType='bin step'
+                  sampleIds={sampleIds}
+                  configChangedHandler={configChangedHandler}
```

Comment:
These names throughout the code (i.e. `configChangeHandler`) would be better renamed to a more callback-style name, i.e. `onConfigChange` or something like that.